### PR TITLE
Remove payload from delete request

### PR DESCRIPTION
**Pull request by warp**
see http://jsonapi.org/format/#crud-deleting

### CODE REVIEW

File: src/middleware/createApiMiddleware.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function handleErrors(response) {
 }
 
 async function handleResponse(response) {
-  const { data, included = [], meta = {} } = await response.json();
+  const { data, included = [], meta = {} } = await response.json()
```
Comment (andyhite) [inline src/middleware/createApiMiddleware.js]: Can you re-add the semi-colon here? It'll break the pre-publish linting otherwise.

**Comment by andyhite** [comment]
Thanks for fixing this. The `delete` method was the only one that I hadn't pretty thoroughly tested outside of unit tests, so I hadn't caught these issues. Just make sure the linter passes (`npm run lint`) and I'll get this merged.

File: src/middleware/createApiMiddleware.js (hunk 1 above)
Comment (warp) [reply src/middleware/createApiMiddleware.js]: ok done

**Comment by andyhite** [comment]
Released v1.1.3 with this fix included.

**Pull request merged**